ddt_request_history/panels/request_history.py

```diff
@@ -27,6 +25,14 @@
 
 DEBUG_TOOLBAR_URL_PREFIX = getattr(settings, 'DEBUG_TOOLBAR_URL_PREFIX', '/__debug__')
 
+
+try:
+    from debug_toolbar.settings import get_config
+    Config = get_config()
+except AttributeError:
+    from debug_toolbar.settings import Config
```

Comment:
Thanks. I think this needs to catch ImportError instead. Also needs to needs to import CONFIG so maybe change this to:

    config = get_config()
except ImportError:
    from debug_toolbar.settings import CONFIG as config